src/site/content/en/blog/aem-with-workbox/index.md

```diff
@@ -0,0 +1,138 @@
+
+```json
+{
+  "name": "WKND Adventures and Travel",
+  "short_name": "WKND Adventures and Travel",
+  "start_url": "/content/wknd/us/en.html",
+  "display": "standalone",
+  "theme_color": "#FFDC00",
+  "background_color": "#FF851B",
+  "orientation": "any",
+  "icons": [
+    {
+      "src": "/content/dam/wknd/pwa-logo.png",
+      "type": "image/png",
+      "sizes": "512x512",
+      "purpose": "any maskable"
+    }
+  ]
+}
+```
+
+The start URL can be different from the default landing page for your domain. By changing the `start_url` parameter, you can send your users directly to the application experience rather than the default page a non-authenticated or new user would be presented. This provides PWA users with a smoother, more app-like experience.
+
+AEM understands that different locales can have a different look and feel. You can configure different properties, colors, and icons for different locales or languages and then synchronize the configuration with the linked pages.
+
+Once the PWA is accessed on the browser, you can right click and inspect to bring up DevTools and view the manifest under the Applications panel.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/xWxP7YILDgejrdtbq0hh.png", alt="A PWA in the DevTools Application panel.", width="800", height="966" %}
+
+### Configure the service worker
+
+You can configure the content to cache and the caching strategy to use.
+
+If you have been using service workers you may be familiar with [caching strategies](https://developers.google.com/web/tools/workbox/modules/workbox-strategies). Caching strategies specify which resources to cache and whether to look for those resources in the cache first, the network first, or in the cache with a network fallback. You can then choose the resources to pre-cache when the service worker is installed. AEM Application service workers implement a [warm cache strategy](https://developers.google.com/web/tools/workbox/guides/advanced-recipes#warm_the_runtime_cache) which means user's experience will not break even if you specify a missing or broken path.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/GY9pDPqKTuux6jR26ZZZ.png", alt="Advanced cache management configuration.", width="800", height="753" %}
+
+In AEM, the term "clientlibs" refers to client-side Libraries: the combination of related JavaScript, CSS, and static resources that have been added to your project that are served to and utilized by the client web browser. You can easily set your client-side libraries to be used offline by specifying those libraries in the user interface.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/9jrYgsiQzPecwYPVdCEw.png", alt="The client side libraries dialog box.", width="800", height="634" %}
```

Comment:
```suggestion
<figure>
  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/9jrYgsiQzPecwYPVdCEw.png", alt="The client side libraries dialog box.", width="800", height="634" %}
</figure>
```

Reply:
More descriptive alt text here as well. From here on out, this will go without saying.